docs/io/fits/index.rst

```diff
@@ -11,8 +11,13 @@ Introduction
 
 The :mod:`astropy.io.fits` package provides access to FITS files. FITS
 (Flexible Image Transport System) is a portable file standard widely used in
-the astronomy community to store images and tables. This subpackage was
-originally developed as PyFITS.
```

Comment:
Not sure if this PR is the good place to remove this sentence. PyFITS is mentioned a few times in the docs, including a few lines below, and I think it's good to state clearly that `io.fits` comes from PyFITS.

Reply:
Just curious about why. That fork was like 14 years ago, so I think that this history should not be the 3rd piece of information that new users learn about `astropy.io.fits`. If there needs to a nod to the history I would put that in an acknowledgements section at the end. PyFITS itself has been deprecated for 8 years and is unlikely to work with modern Python.

Reply:
PyFITS is mentioned in the docs (in this page) so this sentence make it clear what PyFITS is (for people that don't know it) or what it became. It might be time to revisit how we mention it (and link to the [history page](https://docs.astropy.org/en/latest/io/fits/appendix/history.html)) but that deserves its own PR imo.

Reply:
OK, I'll revert for this PR. But there really should be a large followup PR to reflect the fact that PyFITS is now strictly just historical legacy. Searching for PyFITS in the RST docs shows LOTS of just-plain-wrong statements, like: 

The `astropy.io.fits` module is identical to PyFITS but with the names changed.
... There are plans to gradually phase out PyFITS as a stand-alone
module and deprecate it in favor of `astropy.io.fits`.

Basically there is a lot of historical cruft that can only serve to confuse or distract current users.